packages/frontend/src/components/UserSettingsModal/AppearancePanel/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Intent } from '@blueprintjs/core';
+import { ECHARTS_DEFAULT_COLORS } from 'common';
+import React, { FC, useEffect, useState } from 'react';
+import { useApp } from '../../../providers/AppProvider';
+import { useTracking } from '../../../providers/TrackingProvider';
+import SeriesColorPicker from '../../ChartConfigPanel/Series/SeriesColorPicker';
+import {
+    AppearanceColorWrapper,
+    AppearancePanelWrapper,
+    ColorLabel,
+    ColorPalette,
+    Title,
+} from './AppearancePanel.styles';
+
+const AppearanceColor: FC<{
+    color: string;
+    index: number;
+    onChange: (value: string) => void;
+}> = ({ color, index, onChange }) => {
```

Comment:
Lets extract the types and bring them to their own interface, here is fine, but when we have a large number of props it just looks cleaner and it is easier to maintain. 

```
interface AppearanceColorProps {
    color: string;
    index: number;
    onChange: (value: string) => void;
}

const AppearanceColor: FC<AppearanceColorProps> = ({ color, index, onChange }) => {...}

```